### ISSUE TITLE
Error when minrecords is set to 0

### ISSUE THREAD

**Issue by dcadiou** [catalyst/moodle-tool_cohortdatabase#11]
When the plugin setting _minrecords_ (that tells how many records should exists, at least, in the database for the plugin to operate) is set to 0, the plugin operate as if the database table was empty.

An error message is displayed, such as : 

> Failed to sync because the external db returned 0 records and the minimum is 0

This behaviour can be a problem for anybody who want to test the plugin with a table containing only one specific record. (it was my case). A '0' setting could also be used to disable this check for anybody who want that.

**Comment by danmarsden** [comment]
I'm not convinced this is needed.. nothing will happen when records are 0 anyway?  - testing with 1 specific record could be done with a min record setting of 1.

**Comment by dcadiou** [comment]
Hi danmarsden.

thank you for your answer. I agree that this point is not a major point. But, I spent a certain amount of time trying to debug my settings (with a table containing only 1 record). So my proposal is just here to help others not to face the same problem.
 
I just tried with a setting of 1 (it's what I started with, last week), but no, it doesn't work. Running with a minrecords setting to 1 and a table with only 1 record fails. 

In `classes/sync.php`, the line (110) `if ($count > $minrecords) {` make it fails.

The priority is probably to have a minrecords's description in the plugin settings ("..._.if the number of records in the external table is below this number._ ...") in relation with the current behaviour of the plugin. So perhaps my first pull request proposal (https://github.com/catalyst/moodle-tool_cohortdatabase/pull/12) is not the best solution. It's probably better to change only the criteria in line 110 ('>=' instead of '>'). 

I will add a new commit to my pull request.
